src/components/shared/forms/btn.js

```diff
@@ -9,9 +9,9 @@ import { joinClasses } from 'utilities';
 import './styles/btn.css';
 
 export const Btn = (props) => {
-  const {svg, children, className, ...btnProps } = props;
+  const { svg, children, className, primary, ...btnProps } = props;
   return (
-    <button type="button" {...btnProps} className={joinClasses('btn', className)}>
+    <button type="button" {...btnProps} className={joinClasses('btn', className, primary ? 'btn-primary' : 'btn-secondary')}>
```

Comment:
I thought there were three versions of buttons?

Reply:
Nope.  Just two... but with lots of states for each... hover, active, focus, disabled.